### ISSUE TITLE
With Django 1.10.5 something has seriously broken.

### ISSUE THREAD

**Issue by cnobile2012** [jazzband/django-debug-toolbar#905]
Any and I do mean any tab that is clicked shows only a yellow bar at the top of the screen. The background is pretty much the screen you were on except for the yellow bar at the top. Essentially nothing works properly. This started happening as soon as I upgraded to Django 1.10.5. 1.10.4 worked fine.

**Comment by matthiask** [comment]
![screenshot from 2017-01-06 17-20-52](https://cloud.githubusercontent.com/assets/2627/21724420/8c94ec88-d434-11e6-8382-f7e11c376fe5.png)

I can't reproduce this locally. Do you get a Javascript error in the console or somewhere? Are the debug toolbar assets loaded correctly?

**Comment by cnobile2012** [comment]
There is no error anywhere. As I said it all worked perfectly in Django 1.10.4. It is even broken in my Samsung tablet. The 'Hide >>' tab does work, but none of the others function at all.

This is my dev env settings:
```
MIDDLEWARE.append('debug_toolbar.middleware.DebugToolbarMiddleware')

INSTALLED_APPS.append('debug_toolbar')

DEBUG_TOOLBAR_CONFIG = {
    'DEBUG_TOOLBAR_PATCH_SETTINGS': False,
    }
```

My urls.py:
```
if settings.DEBUG:
    # Static media files.
    import debug_toolbar

    urlpatterns += [
        ...
        url(r'^__debug__/', include(debug_toolbar.urls)),
        ]
```

I don't know what else to say. The Django release notes doesn't seem to indicate anything strange.
https://docs.djangoproject.com/en/1.10/releases/1.10.5/#django-1-10-5-release-notes

**Comment by cnobile2012** [comment]
I found the problem finally, though it worked fine in previous versions of Django. It was how the ```urlpatterns``` were configured. I had this in my master ```urls.py```:

    urlpatterns = [
        ...
        url(r'^', include('inventory.sites.urls')),
        ]

When I changed it to:

    urlpatterns = [
        ...
        url(r'^$', include('inventory.sites.urls')),
        ]

everything worked as usual. That URL messed with the ```__debug__``` path but nothing else that came after it. I don't understand why this would have happened since the DDT path is uniquely defined.
